### ISSUE TITLE
Fix tag with spaces

### ISSUE THREAD

**Issue by lapo-luchini** [pinry/pinry#75]
The underlying tag library (and most of pinry logic) seems to be using a comma separator so I figured that tag with spaces should be allowed, but right now they separate just as well (and they show up as commas on edit).

**Comment by lapo-luchini** [comment]
About tag enhancements, would also be nice to have tag aliases.
As a way to say «tag "A" is an alias for tag "B"» (or tag B is the official form of tag A).

**Comment by overshard** [comment]
I agree, this helps with things like plurals, etc. It's not supported by the current tagging system, may be something we just need to completely rework. This is something to think about when working on the tag system.
